### ISSUE TITLE
visions 0.3.0 on pypi pointing to missing requirements-file

### ISSUE THREAD

**Issue by ralphdewit** [dylan-profiler/visions#40]
**Describe the bug**
I cannot pip install visions 0.3.0, it is pointing to a missing requirements-file. I see requirements.txt is currently in the master-branch on Github, but when I 'pip download visions' (0.3.0), there is only a requires.txt residing in 'src/visions.egg-info/'.

Couldn't find it in the closed or open issues, I'll just use 0.2.2 with pandas-profiling for now.

**To Reproduce**
1. pip install visions
2. See error:

```File "/Users/ralphdewit/.pyenv/versions/3.8.0/lib/python3.8/pathlib.py", line 1054, in _opener
        return self._accessor.open(self, flags, mode)
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Expected behavior**
visions 0.3.0 being installed in a virtual environment.

**Comment by ieaves** [comment]
Thanks @ralphdewit, we appreciate your patience on this one. Things are a little hectic at the moment with the Coronavirus but we will try to get a new version pushed out to pypi soon.

**Comment by sbrugman** [comment]
@ralphdewit thanks! Should be solved in 0.4.1 (on Pypi now).
